code/mob/living/carbon/human.dm

```diff
@@ -205,6 +205,8 @@
 
 	src.organHolder = new(src)
 
+	src.ensure_bp_list()
```

Comment:
Nothing else sets the `blood_pressure` var, if it goes missing then just throw out the runtime rather than try to guard against admins doing goofy things in Life() and everything else that accesses it.